python/python/Ice/Communicator.py

```diff
@@ -1,24 +1,53 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 
-__name__ = "Ice"
+from .ObjectAdapter import ObjectAdapter
+from .ImplicitContext import ImplicitContext
+from .Properties import Properties
+from .LoggerI import LoggerI
+from .Logger import Logger
 
-class Communicator(object):
+class Communicator:
```

Comment:
Should we mark it as `@final` like in C++, C# etc.?

Reply:
Fixed. I added the `@final` decorator.